nutanix/data_source_nutanix_subnets.go

```diff
@@ -0,0 +1,477 @@
+package nutanix
+
+import (
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	v3 "github.com/terraform-providers/terraform-provider-nutanix/client/v3"
+	"github.com/terraform-providers/terraform-provider-nutanix/utils"
+)
+
+func dataSourceNutanixSubnets() *schema.Resource {
+	return &schema.Resource{
+		Read:          dataSourceNutanixSubnetsRead,
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
```

Comment:
could you add a input as optional, let's say name?

I think we already have in master a way to get subnets by name, it would be nice if we can add it.